pages/common/views/components/datatable.jsx

```diff
@@ -31,9 +32,9 @@ const Table = ({
           data.map(row => (
             <tr key={row.id}>
               {
-                map(columns, ({ format, accessor }, key) => {
-                  const datum = get(row, accessor || key);
-                  return <td key={key}>{ format ? format(datum, row) : datum }</td>;
+                map(columns, (options, key) => {
+                  const datum = getValue({ row, schema: options, key });
+                  return <td key={key}>{ options.format ? options.format(datum, row) : datum }</td>;
```

Comment:
Name the iterator variable `schema` so you don't need `schema: options` in the arguments?